src/cloudhsm/setup.py

```diff
@@ -0,0 +1,49 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup, find_packages
+
+
+# HISTORY.rst entry.
+VERSION = '1.0.0b1'
```

Comment:
it doesn't match the version in `HISTORY.rst`, the initial release version is `1.0.0b1` by default.